icu4c/source/i18n/vtzone.cpp

```diff
@@ -1217,6 +1217,13 @@ VTimeZone::getOffset(UDate date, UBool local, int32_t& rawOffset,
     return tz->getOffset(date, local, rawOffset, dstOffset, status);
 }
 
+void VTimeZone::getOffsetFromLocal(
+    UDate date, UTimeZoneLocalOption nonExistingTimeOpt,
+    UTimeZoneLocalOption duplicatedTimeOpt,
+    int32_t& rawOffset, int32_t& dstOffset, UErrorCode& status) const {
```

Comment:
put { on a new line on the left

Reply:
why?

Reply:
There are so many C++ function in ICU start that way I thought that is the coding style in ICU.

Reply:
Are you suggesting to apply this to entire file, including a few dozens of existing methods?

Reply:
I'm not trying to change format for now. If we have specific code formatter, I'd prefer to run it globally.

Reply:
ok